STEERING-COMMITTEE.md

```diff
@@ -162,13 +117,157 @@ There are two ways to raise issues to the steering committee for decision:
+
+### Candidate Eligibility
+
+Community members must be eligible to vote in order to stand for election (this
+includes voters who qualify for an exception). Candidates may self-nominate or
+be nominated by another eligible member. The approximate time commitment of a
+Steering member is around 5 hours per week.
+
+To run for a Contributing seat, a candidate must additionally be at least a
+project Member as defined in Roles.md.
+
+To run for an End-User seat, a candidate must be employed by, or own, an
+organization that does not sell products or services derived from Knative.
+Staff with in-house Knative deployments, as well as application developers who
+create applications that run on Knative, are eligible (and encouraged to run)
+for these seats.  Additionally, the candidate must participate in the
+knative-users mailing list and/or community-facing Knative Slack channels.
+
+### Voting Procedure
+
+Elections will be held using a time-limited
+[Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on
+[CIVS](http://civs.cs.cornell.edu/) using the Schulze method. The top
+vote-getters will be elected to the open seats, with the exceptions for company
+representation discussed below. This is the same process used for TOC elections.
+
+### Limitations on Company Representation
+
+No more than two seats may be held by employees of the same organization (or
+conglomerate, in the case of companies owning each other).  Additionally, the two
+End-User seats may not belong to employees of the same organization. If the
+results of an election result in greater than two employees of the same
+organization, the lowest vote getters from any particular employer will be
+removed until representation on the committee is down to two.
+
+If employers change because of job changes, acquisitions, or other events, in a
+way that would yield more than 2 seats being held by employees of the same
+organization, sufficient members of the committee must resign until only two
+employees of the same employer are left. If it is impossible to find sufficient
+members to resign, all employees of that organization will be removed and new
+special elections held. In the event of a question of company membership (for
+example evaluating independence of corporate subsidiaries) a majority of all
+non-involved Steering Committee members will decide.
+
+## Vacancies
+
+In the event of a resignation or other loss of an elected SC member, the
+candidate with the next most votes from the previous election will be offered
+the seat, provided that person otherwise qualifies to join the SC. This process
+will continue until the seat is filled.
+
+In case this fails to fill the seat, a special election for that position will
+be held as soon as possible, unless the regular SC election is less than 7 weeks
+away.  Eligible voters from the most recent election will vote in the special
+election (ie: eligibility will not be redetermined at the time of the special
```

Comment:
I'd change the parenthetical to be inline: "Eligible voters from the most recent election will vote in the special election. Eligibility is not redetermined for the special election."

Reply:
ack